odoo/addons/base/views/res_partner_views.xml

```diff
@@ -208,10 +208,16 @@
                         <group>
                             <field name="function" placeholder="e.g. Sales Director"
                                 attrs="{'invisible': [('is_company','=', True)]}"/>
-                            <field name="phone" widget="phone"/>
-                            <field name="mobile" widget="phone"/>
+                            <div id="base_phone">
```

Comment:
Site note: in odoo we generally use "name" attribute in views instead of ids . Name should also be prefixed if possible, like "partner_form_email", "partner_form_phone", ... it helps understanding it + helps grepping it through odoo .